microvmConfigurations/guivm/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  nixpkgs,
+  microvm,
+  system,
+}:
+nixpkgs.lib.nixosSystem {
+  inherit system;
+
+  modules = [
+    # TODO: Enable only for development builds
+    ../../modules/development/authentication.nix
+    ../../modules/development/ssh.nix
+    ../../modules/development/packages.nix
+
+    microvm.nixosModules.microvm
+
+    ({pkgs, ...}: {
+      networking.hostName = "guivm";
+      # TODO: Maybe inherit state version
+      system.stateVersion = "22.11";
+
+      microvm.hypervisor = "qemu";
+
+      nixpkgs.overlays = [
+        (self: super: {
```

Comment:
As a proposal: let's use `final: prev` instead of `self: super`?

Reply:
The names are replaced.